packages/components/rollup.config.mjs

```diff
@@ -1,35 +1,63 @@
 /* eslint-disable import/no-default-export */
-import multiInput from "rollup-plugin-multi-input";
+import path from "path";
+import { fileURLToPath } from "url";
+import { dirname } from "path";
 import typescript from "@rollup/plugin-typescript";
 import postcss from "rollup-plugin-postcss";
 import commonjs from "@rollup/plugin-commonjs";
 import copy from "rollup-plugin-copy";
+import { nodeResolve } from "@rollup/plugin-node-resolve";
+import postcssimport from "postcss-import";
+import autoprefixer from "autoprefixer";
+import tools from "@csstools/postcss-global-data";
+import presetenv from "postcss-preset-env";
+import multiInput from "rollup-plugin-multi-input";
+import nodePolyfills from "rollup-plugin-polyfill-node";
+import alias from "@rollup/plugin-alias";
 
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
 export default {
-  input: `src/*/index.{ts,tsx}`,
+  input: `src/**/index.{ts,tsx}`,
   plugins: [
-    multiInput(),
+    nodePolyfills(),
+    alias({
+      entries: [
+        {
+          find: /@jobber\/hooks\/(.*)/,
```

Comment:
So that imports such as `import { useAssert } from '@jobber/hooks/useAssert` work properly.